cbt.go

```diff
@@ -1203,6 +1227,16 @@ func doLookup(ctx context.Context, args ...string) {
 		opts = append(opts, bigtable.RowFilter(filters[0]))
 	}
 
+	var stats *bigtable.FullReadStats
+	if includeStats := parsed["include-stats"]; includeStats != "" {
```

Comment:
You could simplify this by allowing a `case ""` that does nothing.

Reply:
Done.